library/psa_its_file.c

```diff
@@ -209,7 +213,12 @@ psa_status_t psa_its_set( psa_storage_uid_t uid,
     }
     if( status == PSA_SUCCESS )
     {
+#if defined(_WIN32)
```

Comment:
Please avoid non-structural use of the preprocessor, with `#ifdef` around things that aren't a standalone C block. Here, I think the best solution would be to define a function-like macro `rename_replace_existing( oldpath, newpath )` with a platform-dependent definition.